contents/docs/advanced/proxy/pomerium.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Using Pomerium as a reverse proxy
+sidebar: Docs
+showTitle: true
+---
+
+import RegionWarning from "../_snippets/region-warning.mdx"
+import ProxyWarning from "../_snippets/proxy-usage-warning.mdx"
+
+<ProxyWarning />
+
+<RegionWarning />
+
+[Pomerium](https://www.pomerium.com/) is an identity-aware proxy that can be used to securely proxy traffic to PostHog. This guide will show you how to configure Pomerium as a reverse proxy for PostHog.
+
+## Basic setup
+
+### Pomerium Zero
+
+The assumption is you've already set up [Pomerium Zero](https://www.pomerium.com/zero).
+
+#### Create an Optional Allow All Policy
+
+1. [Log in to Pomerium Zero](https://console.pomerium.app/).
```

Comment:
I have screenshots I can add here @ivanagas, but I noticed the others had none, so omitted them for the time being.

Reply:
Feel free to add them here and I'll upload them to our CDN and send them back to you